### PR TITLE
Fix decompression and port to standard library lzma.

### PR DESCRIPTION
**Pull request by parona-source**
Tested this by manually compression files in the elog dir with gz, xz and bz2.

### CODE REVIEW

File: elogv [hunk 1]
```diff
@@ -418,18 +413,18 @@ class ElogViewer:
         self.logf_wrap = self.wrap_logf_lines()
         self.show_log()
 
-    def openfile(self, myfile):
+    def open(self, myfile, mode='rt'):
```
Comment (hartwork) [inline elogv:416]: The new method `open`does not seem to use `self`. Would you mind trading `self` for `@staticmethod` to communicate this behavior?

**Review by hartwork — COMMENTED** [review]
@parona-source looks good already, one idea below:

**Comment by hartwork** [comment]
@parona-source thanks! I would release 0.8.2 now unless you have something more that you'd want us to include. What do you think?

**Comment by parona-source** [comment]
Only thing I can think would be to bump it to python3.12 in the ebuild. I haven't noticed anything that would break in python3.12 now that distutils is no more. I've been running elogv with python3.12 while testing it for this merge request.

**Comment by hartwork** [comment]
@parona-source good cool, will do! :+1: Let's go! :smiley:

**Comment by hartwork** [comment]
Python 3.12 done in https://github.com/gentoo/gentoo/commit/d6f28ae1daa76582d2a3d12e1265bef94d48dc19

**Pull request merged**